dashboard/src/components/Catalog/Catalog.test.tsx

```diff
@@ -14,6 +14,7 @@ const defaultChartState = {
   selected: {} as IChartState["selected"],
   deployed: {} as IChartState["deployed"],
   items: [],
+  searchItems: [],
```

Comment:
I'd add a comment saying why we are splitting `items` and `searchItems`. Also, to make it even more clear, I would change `items` for an object:

```
items: {
 all: [],
 filtered: [],
},
```

Reply:
Also, I am not sure this distinction is providing any caching. Note that even if we already have "all" items fetched from before, if the search term is removed, we are fetching all the charts again anyway. That renders the loading wrapper and ignore the current state. Am I wrong?